packages/flutter_driver/lib/src/common/find.dart

```diff
@@ -252,3 +252,39 @@ class ByType extends SerializableFinder {
     return new ByType(json['type']);
   }
 }
+
+
+/// A Flutter driver command that retrieves a semantics id using a specified
+/// finder.
```

Comment:
What happens if the thing identified by finder doesn't own a semantics node?

Reply:
Addressed in https://github.com/flutter/flutter/pull/19047